connector/connect/client/jvm/src/main/scala/org/apache/spark/ml/feature/LabeledPoint.scala

```diff
@@ -0,0 +1,41 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.ml.feature
+
+import org.apache.spark.annotation.Since
+import org.apache.spark.ml.linalg.Vector
+
+/**
+ * Class that represents the features and label of a data point.
+ *
+ * @param label
+ *   Label for this data point.
+ * @param features
+ *   List of features for this data point.
+ */
+@Since("3.5.0")
+case class LabeledPoint(label: Double, features: Vector) {
```

Comment:
can we also move this to mllib-common ?

Reply:
done, but we can not change the since version then